apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -10,6 +10,7 @@ import type { EventNameObjectType } from "@calcom/core/event";
 import { getEventName } from "@calcom/core/event";
 import DestinationCalendarSelector from "@calcom/features/calendars/DestinationCalendarSelector";
 import { FormBuilder } from "@calcom/features/form-builder/FormBuilder";
+import lockedFieldsManager from "@calcom/lib/LockedFieldsManager";
```

Comment:
New lib to let us operate with a managed event type or a children managed event type